src/main/java/at/hannibal2/skyhanni/utils/LocationUtils.kt

```diff
@@ -39,4 +41,7 @@ object LocationUtils {
         val inFov = true // TODO add Frustum "Frustum().isBoundingBoxInFrustum(entity.entityBoundingBox)"
         return noBlocks && notTooFar && inFov
     }
+
+    fun inMiningIsland() = IslandType.GOLD_MINES.isInIsland() || IslandType.DEEP_CAVERNS.isInIsland()
```

Comment:
This belongs in LorenzUtils. LocationUtils is currently only for utils functions around LorenzVec, not for skyblock related functions. We should maybe change this in the future and move all "isinSkyBlock" etc stuff around, but not in this pr.